examples/social-network/config/environments/development.js

```diff
@@ -1,4 +1,25 @@
 export default {
+  server: {
+    cors: {
+      origin: '*',
+      enabled: true,
+
+      headers: [
+        'Accept',
+        'Content-Type'
+      ],
+
+      methods: [
+        'GET',
+        'POST',
+        'PATCH',
+        'DELETE',
+        'HEAD',
+        'OPTIONS'
+      ]
+    }
+  },
+
```

Comment:
I thought it might be useful to show the CORS config in an example project, so I've left this in here for now.